codeforamerica/vita-min
# [Fyst-1224] nc screener flow

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-1224

## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- add new columns + update eligibility rules for nc intakes
- Update form to just show new question the be valid

## How to test?
- Start NC flow
- Verify eligiblity requirements
## Screenshots (for visual changes)

![Screenshot 2024-12-02 at 11 29 02 AM](https://github.com/user-attachments/assets/cc51f0b9-772d-4885-958f-782f4f3e751b)
